docker-compose.yml

```diff
@@ -54,6 +54,37 @@ services:
       - "/var/run/docker.sock:/var/run/docker.sock"
     container_name: logspout
     command: syslog+tls://${ELK_ENDPOINT}
+  
+  cadvisor:
+    restart: on-failure
+    image: gcr.io/google-containers/cadvisor:latest
```

Comment:
use `gcr-io.mirrors.sjtug.sjtu.edu.cn`, otherwise it will take years to pull this image.

Reply:
Never thought about that :((